src/program/managers/event_manager.py

```
@@ -1,5 +1,7 @@
 import os
+import sys
 import threading
+import time
```

Ruff (0.8.0)

`time` imported but unused

Remove unused import: `time`

(F401)